Tests/Microsoft.AppCenter.Crashes.Test.Windows/Microsoft.AppCenter.Crashes.Test.Windows.csproj

```diff
@@ -0,0 +1,77 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\packages\MSTest.TestAdapter.1.3.2\build\net45\MSTest.TestAdapter.props" Condition="Exists('..\packages\MSTest.TestAdapter.1.3.2\build\net45\MSTest.TestAdapter.props')" />
```

Comment:
Other projects use 1.1.18. Probably it isn't a big deal, right?

Reply:
Probably not.  We decided to make the minimum changes, and since there's no major version upgrade, we assume that there's no breaking changes and that we could only get benefits from using the newer version.  That may not be true but if it's not then the package creators aren't obeying semantic versioning :)